This means Kodi no longer finds MySQL:
```
CMake Error at /home/neil/projects/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.0-devel/toolchain/share/cmake-3.7/Modules/FindPackageHandleStandardArgs.cmake:138 (message):
  Could NOT find MySqlClient (missing: MYSQLCLIENT_INCLUDE_DIR)
Call Stack (most recent call first):
  /home/neil/projects/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.0-devel/toolchain/share/cmake-3.7/Modules/FindPackageHandleStandardArgs.cmake:378 (_FPHSA_FAILURE_MESSAGE)
  cmake/modules/FindMySqlClient.cmake:42 (find_package_handle_standard_args)
  cmake/scripts/common/Macros.cmake:392 (find_package)
  CMakeLists.txt:165 (core_optional_dep)
```

--- a/cmake/modules/FindMySqlClient.cmake
+++ b/cmake/modules/FindMySqlClient.cmake
@@ -21,7 +21,7 @@ else()
   set(EXTRA_FIND_ARGS)
 endif()
 
-find_path(MYSQLCLIENT_INCLUDE_DIR mysql/mysql_time.h)
+find_path(MYSQLCLIENT_INCLUDE_DIR NAMES mysql/mysql.h mysql/server/mysql.h)
 find_library(MYSQLCLIENT_LIBRARY_RELEASE NAMES mysqlclient libmysql
                                          PATH_SUFFIXES mysql
                                          ${EXTRA_FIND_ARGS})

